docs/snapshotters/devmapper.md

```diff
@@ -1,11 +1,11 @@
 ## Devmapper snapshotter
 
-Devmapper is a `containerd` snapshotter plugin that stores snapshots in ext4-formatted filesystem images
-in a devicemapper thin pool.
+Devmapper is a `containerd` snapshotter plugin that stores snapshots in Copy-on-Write filesystem images
+in a Device Mapper thin-pool. The devmapper plugin takes advantage of Device Mapper features like [device snapshot support](https://www.kernel.org/doc/Documentation/device-mapper/snapshot.txt).
```

Comment:
The upstream [device-mapper project](http://www.sourceware.org/dm/) and the Linux kernel documentation both appear to use "Device-mapper" or "device-mapper" as the preferred nomenclature.  I've also seen "Device mapper" and "device mapper" elsewhere (such as [Wikipedia](https://en.wikipedia.org/wiki/Device_mapper)).  I have not seen "Device Mapper" anywhere except in the title of articles (where [title case](https://en.wikipedia.org/wiki/Title_case) is used).  The capitalized form ("Device") does seem to mostly appear at the beginning of sentences (following [sentence case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case)).